kubetest2-gke/deployer/utils/util.go

```diff
@@ -27,26 +27,32 @@ import (
 )
 
 var (
-	gkeMinorVersionRegex = regexp.MustCompile(`^(\d\.\d+).*$`)
+	gkeMinorVersionRegex = regexp.MustCompile(`^v?(\d\.\d+).*$`)
 )
 
 // StageGKEBuildMarker stages the build marker to the stage location.
 func StageGKEBuildMarker(version, stageLocation, markerPrefix string) error {
-	m := gkeMinorVersionRegex.FindStringSubmatch(version)
-	var fName string
-	if len(m) < 2 {
-		klog.Warningf("can't find the minor version of %s, defaulting to %s.txt", version, markerPrefix)
-		fName = fmt.Sprintf("%s.txt", markerPrefix)
-	} else {
-		minor := m[1]
-		fName = fmt.Sprintf("%s-%s.txt", markerPrefix, minor)
-	}
-	pushCmd := fmt.Sprintf("gsutil -h 'Content-Type:text/plain' cp - %s/%s", stageLocation, fName)
+	destinationURL := stageLocation + "/" + fileName(version, markerPrefix)
+	pushCmd := fmt.Sprintf("gsutil -h 'Content-Type:text/plain' cp - %s", destinationURL)
 	cmd := exec.RawCommand(pushCmd)
+	if !strings.HasPrefix(version, "v") {
+		version = "v" + version
```

Comment:
adding a `v` prefix breaks existing consumers of this file, please revert this change